src/routes/AppRoutes.tsx

```diff
@@ -22,6 +22,7 @@ const AppRoutes = () => {
         display: 'block',
         boxShadow: '0px 5px 20px 5px rgba(36, 35, 33, 0.17)',
         borderRadius: '20px',
+        padding: '0px!important',
```

Comment:
Space between style and `!important`. Secondary thing: is this `!important` really unavoidable?